mdx.opam

```diff
@@ -10,7 +10,7 @@ doc:          "https://realworldocaml.github.io/mdx/"
 build: [
  ["dune" "subst"] {pinned}
  ["dune" "build" "-p" name "-j" jobs]
- ["dune" "runtest" "-p" name] {with-test}
+ ["make" "test"] {with-test}
```

Comment:
I don't understand this change -- why is this needed?

Reply:
`dune runtest -p mdx` doesn't work because some of our tests rely on a different local package which doesn't get built because of the `-t` option.

`dune runtest` alone doesn't work because `ocaml-mdx` depends on `ocaml-mdx-test` and we can't specify dependencies on executables in dune.

Finally running `dune build @install @runtest` can fail because of a race condition ie some tests are run before `ocaml-mdx-test` is built.

Since we were using `make test` to run the test in the CI I switched to that. I also made it clear in the makefile that `dune runtest` can only be reliably run after `dune build`.

Reply:
I tried other solutions with dune but couldn't get any of them working and always ended up with a race condition, I think using the Makefile to wrap dune commands the way we want it is okay but if there's any issue with that we can look for an alternative!

Reply:
Can you report this upstream (and point to that PR to see it's annoying for us)? It would be nice if we didn't have to add hacks to make the tests work.

Reply:
We have a related issue on dune already, I'll update it!

Reply:
Did that here: https://github.com/ocaml/dune/issues/2470#issuecomment-537998717